www/front_src/src/Resources/Graph/Performance/Lines.tsx

```diff
@@ -7,8 +7,17 @@ import { fade } from '@material-ui/core';
 
 import { fontFamily } from '.';
 import formatMetricValue from './formatMetricValue';
+import { Line as LineModel } from './models';
 
-const getGraphLines = ({ lines, base }): Array<JSX.Element> => {
+interface GraphLinesProps {
+  lines: Array<LineModel>;
+  base?: number;
+}
+
+const getGraphLines = ({
+  lines,
+  base,
```

Comment:
```suggestion
  base = 1000,
```
Same default prop value as `www/front_src/src/Resources/Graph/Performance/formatMetricValue/index.ts`